### DISCUSSION TITLE
RAD12 support

### DISCUSSION THREAD

**Discussion by siegmundertle** [pashagolub/postgresdac#13] [Ideas]
Hi,
do you have planned to support RAD12 in near future?
RAD12 is significantly better than RAD11, especially in C++ personality.

I tried to compily latest 28er version, but the compile failed.
Thahk you!

**Comment by pashagolub** [comment]
Hi Siegmund! 

Yeap, on my todo list :)

**Reply by pashagolub** [reply]
@siegmundertle  Would you please check if it's working for you? #14
